src/index.html

```diff
@@ -0,0 +1,527 @@
+              </p>
+              <p>
+                  What would become our observable universe mushroomed in size at least a trillion trillionfold — from a submicroscopic speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.
+              </p>
+              <p>
+                  Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”
+              </p>
+              <p>
+                  Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.
+              </p>
+              <p>
+                  So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+              </p>
+              <p>
+                  We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.
+              </p>
+              <p>
+                  “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+              </p>
+            </div>
+      </article>
+    </main>
+
+    <aside>
+      <h2>Editor's Picks</h2>
+
+      <div>
+        <span class="img">
+          <img src="../img/editor-1.jpg" alt="editor pick 1">
+        </span>
+        <span class="title">
+            He Wanted a Little Charm Downtown — on a Budget. Which Place Did He Choose?
+        </span>
+      </div>
+
+      <div>
+        <span class="img">
+          <img src="../img/editor-2.jpg" alt="editor pick 2">
+        </span>
+        <span class="title">
+            Was the Gap Ever Cool? A Look at 50 Years in Denim and Khaki
+        </span>
+      </div>
+
+      <div>
+        <span class="img">
+          <img src="../img/editor-3.jpg" alt="editor pick 3">
+        </span>
+        <span class="title">
+            The Sly Superpower of ‘Hustlers’
+        </span>
+      </div>
+    </aside>
+
+    <section class="bottom-sec">
+      <div class="more-in-category">
+        <div class="section-title">More in Space & Cosmos</div>
```

Comment:
Why is this not a heading element? :thinking: 

Consider using `h2-h6` element here instead. `div` has no semantic meaning and this part of the section is obviosly the **heading** of the section.

> Validate your HTML markup with https://validator.w3.org/#validate_by_upload.

Reply:
Resolved.